nautobot/core/templates/search.html

```diff
@@ -65,3 +65,10 @@ <h3 class="text-muted text-center">No results found</h3>
         </div>
     {% endif %}
 {% endblock %}
+
+{% block javascript %}
+    {{ block.super }}
+    <script>
+        var clipboard = new ClipboardJS('.btn');
```

Comment:
Remind me, does this mean it applies to all `btn` classes on the page? If so, does it interfere with the operation of the `Search` or `See All Results/Refine Search` buttons on this page?

Reply:
It does not based on my manual testing. Also this block exists in `object_list.html` and it does not seem to interfere with any other buttons (Edit, Delete, etc) on that page either.